.eslintrc.json

```diff
@@ -0,0 +1,13 @@
+{
+  "parser": "babel-eslint",
+  "extends": ["airbnb"],
+  "env": {
+    "browser": true,
+    "node": true
+  },
+  "rules": {
+    "no-console": "off",
+    "comma-dangle": "off",
```

Comment:
Is this how this rule should be used? On the documentation I see stuff like this:

`"comma-dangle": ["error", {
        "arrays": "never",
        "objects": "never",
        "imports": "never",
        "exports": "never",
        "functions": "ignore"
    }]`

Reference: https://eslint.org/docs/rules/comma-dangle